packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -61,60 +79,64 @@
 .label-value {
   inset-block-end: calc(100% + 0.5em);
   font-weight: var(--calcite-font-weight-bold);
-  color: var(--calcite-color-text-1);
+  color: var(--calcite-meter-value-text-color, var(--calcite-color-text-1));
 }
 
 .label-range {
   @apply text-color-3;
+  color: var(--calcite-meter-range-text-color, var(--calcite-color-text-3));
   inset-block-start: calc(100% + 0.5em);
 }
 
 .unit-label {
-  @apply text-color-3 font-medium;
+  color: var(--calcite-meter-unit-text-color, var(--calcite-color-text-3));
+  font-weight: var(--calcite-font-weight-medium);
 }
 
 .label-value .unit-label {
-  @apply text-color-2 font-bold;
+  color: var(--calcite-meter-unit-text-color, var(--calcite-color-text-2));
+  font-weight: var(--calcite-font-weight-bold);
 }
 
 .fill {
-  @apply block absolute duration-150 ease-in-out bg-brand z-default;
-  inset-inline-start: var(--calcite-meter-space-internal);
-  inset-block: var(--calcite-meter-space-internal);
-  border-radius: var(--calcite-meter-height-internal);
-  max-inline-size: calc(100% - (var(--calcite-meter-space-internal) * 2));
-  min-inline-size: calc(var(--calcite-meter-height-internal) - (var(--calcite-meter-space-internal) * 2));
+  @apply block absolute duration-150 ease-in-out z-default;
+  inset-inline-start: var(--calcite-internal-meter-space);
+  inset-block: var(--calcite-internal-meter-space);
+  border-radius: var(--calcite-meter-corner-radius, 9999px);
+  max-inline-size: calc(100% - (var(--calcite-internal-meter-space) * 2));
+  min-inline-size: calc(var(--calcite-internal-meter-height) - (var(--calcite-internal-meter-space) * 2));
+  background-color: var(--calcite-meter-fill-color, var(--calcite-color-brand));
   transition-property: inline-size, background-color, box-shadow;
 }
 
 .fill-danger {
-  background-color: var(--calcite-color-status-danger);
+  background-color: var(--calcite-meter-fill-color, var(--calcite-color-status-danger));
 }
 
 .fill-success {
-  background-color: var(--calcite-color-status-success);
+  background-color: var(--calcite-meter-fill-color, var(--calcite-color-status-success));
 }
 
 .fill-warning {
-  background-color: var(--calcite-color-status-warning);
+  background-color: var(--calcite-meter-fill-color, var(--calcite-color-status-warning));
 }
 
 .solid .fill {
   inset-block: 0;
   inset-inline-start: 0;
   max-inline-size: 100%;
-  min-inline-size: calc(var(--calcite-meter-height-internal));
-  box-shadow: 0 0 0 1px var(--calcite-color-brand);
+  min-inline-size: calc(var(--calcite-internal-meter-height));
+  box-shadow: 0 0 0 1px var(--calcite-meter-fill-color, var(--calcite-color-brand));
 }
 
 .solid .fill-danger {
```

Comment:
IMO the danger, success, warning classes need to be lifted up into the `:host` classes so that these box-shadow and fill styles only need to be set as props once and then reassigned as CSS vars.

Something like...

```
:host {
  --calcite-meter-fill-color: var(--calcite-color-brand);
  --calcite-meter-box-shadow: 0 0 0 1px var(--calcite-meter-fill-color);
}

:host(.danger) {
  --calcite-meter-fill-color: var(--calcite-color-status-danger);
}
...

.solid .fill {
  ...
  background-color: var(--calcite-meter-fill-color);
  box-shadow: var(--calcite-meter-box-shadow);
```

Reply:
I think we've been avoiding setting any classes directly on `Host`. We could still alias the overrides and only apply once to each `.fill` style

```
:host {
  --calcite-meter-fill-color: var(--calcite-color-brand);
}

.danger {
  --calcite-meter-fill-color: var(var(--calcite-meter-fill-color, --calcite-color-status-danger));
}

.fill {
  background-color: var(--calcite-meter-fill-color);
}
``` 
WDYT?

Reply:
Is there an alternative to setting classes directly on the Host? I think we've been avoiding that. But we can definitely re-alias instead of setting individual roles. Will take a look.

Reply:
I think this is a use case for an internal alias, WDYT? We've had a convention of not setting classes publicly on `Host`, and this seems to align with setting the publicly documented css prop on `Host`. Works well for the theming overrides and defaults.

```
:host {
  --calcite-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-brand));
}

.fill {
  --calcite-internal-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-brand));
}

.fill-danger {
  --calcite-internal-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-status-danger));
}

.fill-success {
  --calcite-internal-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-status-success));
}

.fill-warning {
  --calcite-internal-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-status-warning));
}

.fill {
  background-color: var(--calcite-internal-meter-fill-color);
}

.solid .fill {
  box-shadow: 0 0 0 1px var(--calcite-internal-meter-fill-color);
}

```

Reply:
Above works to provide theming overrides and defaults, going to merge in and we can change later if needed - it's `internal` and any changes won't affect the public variable we introduce here.